### PR TITLE
Update rover-sitlmavproxy-tutorial.rst

### PR DESCRIPTION
**Pull request by karthikdesai**

### CODE REVIEW

File: dev/source/docs/rover-sitlmavproxy-tutorial.rst [hunk 1]
```diff
@@ -57,7 +57,11 @@ Starting
 **Right-click** on the map at your target destination, select **Fly
 to**, and enter an altitude (which is ignored). Rover will change to
 ``GUIDED`` mode, drive to the location, and then pause (still in GUIDED
-mode).
+mode). If there is no response, arm the rover by entering the following on
```
Comment (khancyr) [inline dev/source/docs/rover-sitlmavproxy-tutorial.rst]: we should always arm before using guided mode !

**Comment by karthikdesai** [comment]
Yup! Okay will push commit to say that.
I didn't check the params. My rover refused to obey until it was armed. If it is for everyone then yeah should be armed for sure.. Thought it was just me xd

**Comment by karthikdesai** [comment]
@khancyr good to go now?

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**